apis/v1alpha1/webhook.go

```diff
@@ -161,3 +165,19 @@ func (r *CrdbCluster) ValidateCockroachVersion() error {
 
 	return nil
 }
+
+// ValidateVolumeMode validates that the volumeMode of the pvc is set to filesystem.
+func (r *CrdbCluster) ValidateVolumeMode() error {
+	claim := r.Spec.DataStore.VolumeClaim
+	if claim == nil {
+		return nil
+	}
+	if claim.PersistentVolumeClaimSpec.VolumeMode == nil {
+		return fmt.Errorf("you have not provided pvc.volumeMode value.")
+	}
+	if *claim.PersistentVolumeClaimSpec.VolumeMode != v1.PersistentVolumeFilesystem {
+		return fmt.Errorf(
+			"you have not provided unsupported pvc.volumeMode, currently only Filesystem is supported.")
```

Comment:
correction: rm "not"; should be "you have provided unsupported..."

Reply:
Thanks for the catch. I can't English... I fixed it in an update.